Networking/WebSockets/WebSocketProtocol.hh

```diff
@@ -413,7 +436,13 @@ public:
                 // invalid reserved bits / invalid opcodes / invalid control frames / set compressed frame
                 if ((rsv1(frame) && !setCompressed(user)) || rsv23(frame) || (getOpCode(frame) > 2 && getOpCode(frame) < 8) ||
                     getOpCode(frame) > 10 || (getOpCode(frame) > 2 && (!isFin(frame) || payloadLength(frame) > 125))) {
-                    forceClose(user, "cs-1");
+#ifdef COUCHBASE_forceClose
+                    std::stringstream ss;
+                    ss << "[frame=" << frame << "]";
```

Comment:
What will be printed as frame here? I think we should print out the value that we could track back what is failing here.